### PR TITLE
Scss.animation.theming

### PR DESCRIPTION
**Pull request by jeremie-lucca**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://396-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/utilities/animations/_sliding.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @keyframes slideIn {
```
Comment (lucienbertin) [inline packages/scss/src/utilities/animations/_sliding.scss]: slideIn seems to be a shortcut for slideInTop. is it on purpose ?

Comment (lucienbertin) [reply packages/scss/src/utilities/animations/_sliding.scss]: by that i mean we got
- slideIn
- slideInLeft
- slideInRight
- slideInBottom

but no slideInTop

Comment (sanlucca) [reply packages/scss/src/utilities/animations/_sliding.scss]: Yes, as Top was seen as default

**Pull request merged**